app/ldap.go

```diff
@@ -52,7 +52,7 @@ func (a *App) GetLdapGroup(ldapGroupID string) (*model.Group, *model.AppError) {
 		}
 	} else {
 		ae := model.NewAppError("GetLdapGroup", "ent.ldap.app_error", nil, "", http.StatusNotImplemented)
-		mlog.Error(fmt.Sprintf("%v", ae.Error()))
+		mlog.Error("Unable to use ldap", mlog.String(ldapGroupID))
```

Comment:
@vdepatla, the behaviour change here is unexpected: we're no longer logging the error at all. Can we include it?
```suggestion
		mlog.Error("Unable to use ldap", mlog.String("ldap_group_id", ldapGroupID), mlog.Err(ae))
```